docs/pages/deploy-a-cluster/helm-deployments/aws.mdx

```diff
@@ -394,20 +438,20 @@ podSecurityPolicy:
 Install the chart with the values from your `aws-values.yaml` file using this command:
```

Comment:
Worth noting that sometimes the values file is called `aws-values.yaml` and sometimes it's `values.yaml`.

Reply:
Thanks, they're now all `aws-values.yaml`